apps/cookbook/src/styles.scss

```
@@ -45,3 +45,11 @@ html {
 *:after {
   box-sizing: inherit;
 }
+
+// The <cookbook-code-viewer> component prefixes the class attribute with "language-" on <code> elements.
+// Do not target those.
+code:not([class*='language-']) {
+  border-radius: 4px;
+  background-color: var(--kirby-semi-light, #e2e2e2);
```

We usually don't add fallbacks when referring our css vars - please us our design tokens instead
```suggestion
  background-color: get-color('semi-light');
```